release-tools/Makefile.dist.install.image

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Installation variables
+DESTDIR :=
+DEFAULTSDIR := /usr/share/
+PROJECT_DIR := clear-containers
+IMAGE := @IMAGE@
+
+DESTSYSCONFDIR := $(abspath $(DESTDIR)/$(DEFAULTSDIR)/$(PROJECT_DIR))
+
+
+IMAGE_DEST := $(abspath $(DESTSYSCONFDIR)/$(IMAGE))
+
+
+install:
+	install -D --owner root --group root --mode 0644 $(IMAGE) $(IMAGE_DEST)
+	ln -sf $(IMAGE_DEST) $(DESTSYSCONFDIR)/clear-containers.img
```

Comment:
Should this be?:

```
ln -sf $(IMAGE_DEST) $(IMAGE)
```

Reply:
No, it should be to comply with our current symlink pattern to point latest installed image to DESTSYSCONFDIR/clear-containers.img

Reply:
ok. I'd create a top-level variable for this though to make it more obvious how to switch it if it goes into kata.

Reply:
Added variabl.